src/app/core/component/graph-display/graph-display.component.ts

```diff
@@ -246,8 +308,10 @@ export class GraphDisplayComponent implements AfterContentInit, OnChanges {
     }
   }
 
-  triggerRerendering() {
-    this.graphNativeElement.completeRender();
+  triggerRerendering(forceUpdateTemplates = false) {
+    if (this.graphNativeElement) {
+      this.graphNativeElement.completeRender(forceUpdateTemplates);
+    }
   }
 
   patternListExpanded(patternLang: PatternLanguage) {
```

Comment:
Is this an event handler? If so, why doesn't it consume an event?
A function does something, i.e., a name should indicate this. What is it in this case?

Reply:
It handles the event, which occurs after the list of patterns is expand.